_sass/testimonials.scss

```
@@ -107,25 +112,31 @@
     -moz-user-select: none;
     -ms-user-select: none;
     -webkit-user-select: none;
     user-select: none;
-    line-height: 28px;
+    display: flex;
+    align-items: center;
 
+    svg {
+        width: 100%;
+        height: 100%;
+        fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)